Hcal/include/Hcal/Event/HcalHit.h

```diff
@@ -111,6 +111,38 @@ class HcalHit : public ldmx::CalorimeterHit {
    */
   int getAmplitudeNeg() const { return amplitudeNeg_; }
 
+  /**
+   * Get the position of the hit
+   * @return position
+   */
+  double getPosition() const { return position_; }
+
+  /**
+   * Get if bar is oriented in X
+   * @return orientationX
+   */
+  int getOrientationX_() const { return orientationX_; }
```

Comment:
On a similar note, it is probably nice if these are named in a more descriptive manner. E.g. something like "is oriented in x" etc so the meaning is obvious to people that are new to the code